aws/logs_monitoring/lambda_function.py

```diff
@@ -66,8 +70,14 @@ def datadog_forwarder(event, context):
     if DD_ADDITIONAL_TARGET_LAMBDAS:
         invoke_additional_target_lambdas(event)
 
-    parsed = parse(event, context)
-    enriched = enrich(parsed)
+    # set the prefix for cache layer
+    if not cache_layer.prefix:
```

Comment:
Could we instead create the CacheLayer with the proper prefix directly here ? and pass the prefix to Caches in `CacheLayer.__init__ `